pkg/curl/data_request.go

```diff
@@ -57,6 +58,7 @@ type requestDataSourceModel struct {
 	URI                types.String `tfsdk:"uri"`
 	HTTPMethod         types.String `tfsdk:"http_method"`
 	Data               types.String `tfsdk:"data"`
+	Headers            types.String `tfsdk:"headers"`
```

Comment:
This probably should be a map

so it can be used as `Header1: Value1, Header2: Value2`